### PR TITLE
Basic build and upload for dashboad

### PR DESCRIPTION
**Pull request by aaronchongth**
## What's new

Docker images for general use
- [x] dashboard
- [x] api-server

## Self-checks

- [ ] I have prototyped this new feature (if necessary) on Figma 
- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

File: .github/dashboard/Dockerfile
Comment (koonpeng) [inline .github/dashboard/Dockerfile:1]: can you also add the init scripts to inject global vars.

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -20,12 +20,29 @@ jobs:
...
+          context: .github/dashboard
+      - name: Build and push api-server
+        uses: docker/build-push-action@v5
+        with:
+          push: true
+          build-args: |
+            BASE_IMAGE=ghcr.io/${{ github.repository }}/minimal-rmf
+            ROS_DISTRO=jazzy
+          tags: ghcr.io/${{ github.repository }}/api-server
+          context: .github/api-server
```
Comment (koonpeng) [inline .github/workflows/nightly.yml:45]: There should be a dependency to ensure the latest `minimal-rmf` is used.

File: README.md [hunk 2]
```diff
@@ -6,15 +6,40 @@
...
+# Quick start with docker
+
+Start the dashboard with host network access. The dashboard will then accessible on `localhost:3000` by default.
+
+```bash
+docker run \
+  --network host \
+  -it \
+  ghcr.io/open-rmf/rmf-web/dashboard:latest
+```
```
Comment (koonpeng) [inline README.md:27]: add `--rm`

File: README.md [hunk 3]
```diff
@@ -6,15 +6,40 @@
...
 - [API server](packages/api-server)
 - [API client](packages/api-client)
 - [Dashboard](packages/dashboard)
 - [Configuration](#configuration)
 - [Contribution guide](#contribution-guide)
-- [Roadmap](https://github.com/open-rmf/rmf-web/wiki/Open-RMF-Web-Dashboard)
+- [Roadmap](https://github.com/orgs/open-rmf/projects/10)
 
-# Getting started
+# Quick start with docker
```
Comment (koonpeng) [inline README.md:18]: Would be good to put a link to rmf demos.

File: README.md [hunk 4]
```diff
@@ -56,7 +81,7 @@ You may also install dependencies for only a subset of the packages
 pnpm install -w --filter <package>...
 ```
 
-### Launching
+### Launching for development
```
Comment (koonpeng) [inline README.md:84]: Add a link to rmf demos setup.

**Review by koonpeng — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (aaronchongth) [reply README.md:84]: https://github.com/open-rmf/rmf-web/pull/994/commits/31b6489a5c6b69a38900f3fb52dfdff5d6a43eaa

File: README.md (hunk 3 above)
Comment (aaronchongth) [reply README.md:18]: https://github.com/open-rmf/rmf-web/pull/994/commits/31b6489a5c6b69a38900f3fb52dfdff5d6a43eaa

File: README.md (hunk 2 above)
Comment (aaronchongth) [reply README.md]: good catch, thanks, https://github.com/open-rmf/rmf-web/pull/994/commits/31b6489a5c6b69a38900f3fb52dfdff5d6a43eaa

File: .github/dashboard/Dockerfile
Comment (aaronchongth) [reply .github/dashboard/Dockerfile:1]: https://github.com/open-rmf/rmf-web/pull/994/commits/3d322bf3f98bdf18c31e6de0f2f15b5b6d2e0f9a

**Review by koonpeng — APPROVED** [review]

**Pull request merged**